samples/user-interface/haptics/README.md

```diff
@@ -0,0 +1,115 @@
+Haptic Sampler
```

Comment:
typo? Should it be "Samples"?

Reply:
This has always been called "Sampler" and I always wondered why. Let's rename it to samples.